pkg/names/names_test.go

```diff
@@ -0,0 +1,46 @@
+package names
+
+import (
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("InitRelationshipLabels", func() {
+	Context("When env vars are empty", func() {
```

Comment:
We need to add unit test for env vars not present to ensure that is still working and partial empty and partial not empty.